packages/region/src/services/region-module.ts

```diff
@@ -325,3 +414,25 @@ export default class RegionModuleService<
     }
   }
 }
+
+// microORM complains if undefined fields are present in the passed data object
+const removeUndefined = <T extends Record<string, any>>(obj: T): T => {
```

Comment:
maybe just put a todo here in order to not forget just in case 👍

Reply:
@adrien2p I have a todo in my notion, no worries :) Once it's merged I'll open a PR to clean it up

Reply:
🚀